README.md

```diff
@@ -43,6 +182,103 @@ See [deployment](./CONTRIBUTING.md#Deployment) for how to deploy the contract.
 | mainnet | deployer | [`0x5a5e124ea1f3fc5fcfae3c198765c3b4c8d72c7236ae97ef6e5a9bc7cfda549c`](https://explorer.aptoslabs.com/account/0x5a5e124ea1f3fc5fcfae3c198765c3b4c8d72c7236ae97ef6e5a9bc7cfda549c?network=mainnet) |
 | mainnet | aux      | [`0xbd35135844473187163ca197ca93b2ab014370587bb0ed3befff9e902d6bb541`](https://explorer.aptoslabs.com/account/0xbd35135844473187163ca197ca93b2ab014370587bb0ed3befff9e902d6bb541?network=mainnet) |
 
-## Contributing
+## Deployer
```

Comment:
I'd actually prefer to leave this in the deployer README

Reply:
why an exception for deployer if we're consolidating? isn't it the same logic for `aux-ts`?

Reply:
oh true, maybe all should go in there? Could rename it to DEVELOPMENT

Reply:
I think just to distinguish between what someone would do as a dev vs. a user

Reply:
i moved it to contributing